.github/CONTRIBUTING.md

```diff
@@ -111,7 +153,7 @@ make format_diff
 
 This is especially useful when you have made changes to a subset of the project and want to ensure your changes are properly formatted without affecting the rest of the codebase.
 
-### Linting
+#### Linting
 
 Linting for this project is done via a combination of [Black](https://black.readthedocs.io/en/stable/), [isort](https://pycqa.github.io/isort/), [flake8](https://flake8.pycqa.org/en/latest/), and [mypy](http://mypy-lang.org/).
```

Comment:
`Isort` and `flake8` tests run by `ruff`, as per `Makefile`

> lint lint_diff:
> 	./scripts/check_pydantic.sh .
> 	./scripts/check_imports.sh
> 	poetry run ruff .
> 	poetry run black $(PYTHON_FILES) --check
> 	poetry run mypy $(PYTHON_FILES)

So, maybe we should reference only `ruff`?

Reply:
good call: https://github.com/langchain-ai/langchain/pull/1203#discussion_r1113701076